### PR TITLE
feat: show current mode on hover

### PR DESCRIPTION
**Pull request by NiceAesth**
Helps alleviate some confusion for the system icon.

### CODE REVIEW

**Comment by NiceAesth** [comment]
It seems to work for me as-is

**Comment by NiceAesth** [comment]
Ah, I see. Will look into it now.

**Comment by NiceAesth** [comment]
While working on this, I did find that the way `SoundDark_mode` works is odd as it's always the opposite of whatever is displayed. Did not change that behaviour in this PR.

File: src/services.js [hunk 1]
```diff
@@ -50,20 +50,34 @@ self.addEventListener('DOMContentLoaded', function () {
         }
     }
 
+    function GetTitle(mode) {
```
Comment (NiceAesth) [inline src/services.js:53]: This became duplicated as I cannot set the title in the switch above as the page is not yet loaded in that context

**Comment by michioxd** [comment]
> While working on this, I did find that the way `SoundDark_mode` works is odd as it's always the opposite of whatever is displayed. Did not change that behaviour in this PR.

I think you can put those check `SoundDark_mode` value inside `GetTitle` function. In the append SoundDark menu to the context menu you can do like this:

```js
`...
title="${GetTitle()}"
...`
```

This will get new state during rendering.
Then put your title change function with `GetTitle` value to the end of inside toggle mode function.

**Pull request merged**